CHANGELOG.md

```diff
@@ -6,11 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-### Fixed
-
-- **CUMULUS-1698**
-  - Change variable `saml_launchpad_metadata_path` to `saml_launchpad_metadata_url` in the `tf-modules/cumulus` Terraform module.
-  - Updated `@cumulus/api/launchpadSaml` to download launchpad IDP metadata from configured location when the metadata in s3 is not valid, and to work with updated IDP metadata and SAML response.
+## [v1.18.0] 2020-02-03
```

Comment:
Didn't we deprecate a bunch of stuff? Or is that not merged yet? If we did we should have a list of deprecations here.

Reply:
Doesn't look like Mark added deprecations notes to the changelog. I'll try and find them all.

Reply:
Oh, he did. They were just under `### Changed`. Moved to a `### Deprecated` section.

Reply:
Nice, thank you!